package.json

```diff
@@ -52,12 +52,14 @@
     "semver": "~5.6.0"
   },
   "scripts": {
-    "test": "rimraf dist/* && npm run prepare && istanbul instrument dist/bundle.min.js > dist/bundle.instrumented.js && node test/run-tests.js",
+    "test": "rimraf dist/* && npm run build -- --input=main.tests.js && npm run minify && node test/run-tests.js",
     "coverage": "istanbul cover test/run-tests.js",
     "coveralls": "coveralls < coverage/lcov.info",
     "lint": "eslint lib",
     "lint-fix": "eslint --fix lib",
-    "prepare": "rollup -c && cd dist && cross-env TERSER_NO_BUNDLE=1 ../bin/uglifyjs bundle.js -mc --source-map 'content=bundle.js.map,includeSources=true,url=bundle.min.js.map' -o bundle.min.js",
+    "build": "rollup -c",
+    "minify": "cd dist && cross-env TERSER_NO_BUNDLE=1 ../bin/uglifyjs bundle.js -mc --source-map 'content=bundle.js.map,includeSources=true,url=bundle.min.js.map' -o bundle.min.js",
+    "prepare": "npm run build && npm run minify",
```

Comment:
Can you leave these changes out? I'm working on making the scripts better, but haven't committed it.

Reply:
The package.json changes I mean

Reply:
Unfortunately no ☹️
We need somehow detect that we running tests and then build Terser with this APIs.. PR can wait until you push your changes and then I'll fix conflicts

Reply:
No, I'll handle the conflicts, don't worry!